apps/system/js/app_core.js

```diff
@@ -59,7 +59,8 @@
           'ActivityWindowManager',
           'PermissionManager',
           'Rocketbar',
-          'ActivityHandler'
+          'ActivityHandler',
+          'TrackingNotice'
```

Comment:
Did you try having TrackingNotice lazy loaded by the browser? It makes me sad we are putting TrackingNotice on the same level as thinks like Rocketbar and Activity handler. Better would be to have Browser check the `privacy.trackingprotection.shown` setting, and only lazyload the tracking notice module if necessary. But I realize that is asking a lot for such a short deadline, so I won't block the review on that.